packages/message-kit/src/lib/core.ts

```
@@ -67,43 +64,33 @@ export type Context = {
   getMemoryKey(): string;
   sendTo(message: string, receivers: string[]): Promise<void>;
-  getLastMessageById(reference: string): Promise<any>;
   reply(message: string): Promise<void>;
   dm(message: string): Promise<void>;
   send(
     message: string | Reply | Reaction | RemoteAttachment | Attachment,
     contentType?: ContentTypeId,
     targetConversation?: V2Conversation,
   ): Promise<void>;
-  getConversationKey(): string;
   awaitedHandler: ((text: string) => Promise<boolean | void>) | undefined;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)